src/control/cmd/dmg/pretty/pool.go

```diff
@@ -190,15 +214,28 @@ func printListPoolsResp(out io.Writer, resp *control.ListPoolsResp) error {
 		fmt.Fprintln(out, "no pools in system")
 		return nil
 	}
+	var upgrade bool = false
```

Comment:
(style) In Go, it is considered more idiomatic to use one of the following styles:
```suggestion
	upgrade := false
```
```suggestion
	var upgrade bool
```
The second style leverages the [zero value](https://go.dev/tour/basics/12) of the bool type.